src/store/website-status-context.tsx

```diff
@@ -26,30 +26,26 @@ export const WebsiteStatusProvider = ({ children }: WebsiteStatusProviderProps)
     const WEBSITE_STATUS_COUNTRY_KEY = 'website_status'
     const COOKIE_EXPIRY_DAYS = 7
 
-    const [website_status, setWebsiteStatus] = useCookieState(WEBSITE_STATUS_COUNTRY_KEY, {
+    const [, setWebsiteStatus] = useCookieState(WEBSITE_STATUS_COUNTRY_KEY, {
```

Comment:
@fasihali-deriv Why do we need the comma here? 👀

Reply:
Actually I am destructuring an array here and I only need second index that's why I skipped first index by just adding comma.